packages/material-ui/src/locale/index.ts

```diff
@@ -1927,59 +1927,78 @@ export const viVN: Localization = {
 
 export const zhCN: Localization = {
   props: {
-    // MuiBreadcrumbs: {
-    //   expandText: 'Show path',
-    // },
+    MuiBreadcrumbs: {
+      expandText: '展开',
+    },
     MuiTablePagination: {
-      // getItemAriaLabel: (type) => {
-      //   if (type === 'first') {
-      //     return 'Go to first page';
-      //   }
-      //   if (type === 'last') {
-      //     return 'Go to last page';
-      //   }
-      //   if (type === 'next') {
-      //     return 'Go to next page';
-      //   }
-      //   // if (type === 'previous') {
-      //   return 'Go to previous page';
-      // },
+      getItemAriaLabel: (type) => {
+        if (type === 'first') {
+          return '第一页';
+        }
+        if (type === 'last') {
+          return '最后一页';
+        }
+        if (type === 'next') {
+          return '下一页';
+        }
+        return '上一页';
+      },
       labelRowsPerPage: '每页行数:',
       labelDisplayedRows: ({ from, to, count }) =>
-        `${from}-${to} 的 ${count !== -1 ? count : `超过 ${to}`}`,
+        `第 ${from} 项到第 ${to} 项，${count !== -1 ? `共 ${count} 项` : `至少 ${to} 项`}`,
     },
     MuiRating: {
-      getLabelText: (value) => `${value} 星${value !== 1 ? '星' : ''}`,
-      // emptyLabelText: 'Empty',
+      getLabelText: (value) => `${value} 星`,
+      emptyLabelText: '无标签',
     },
     MuiAutocomplete: {
-      clearText: '明确',
-      closeText: '关',
+      clearText: '清空',
+      closeText: '关闭',
       loadingText: '载入中…',
```

Comment:
we should use proper period. `载入中。。。`

Reply:
`。。。` is not the proper period IMO. `……` or `......` is better.

Reply:
https://github.com/ant-design/ant-design/blob/master/components/locale/zh_TW.tsx#L49

```suggestion
      loadingText: '载入中...',
```

Reply:
Sorry you are right, using `……` please. 
Also for not giving the suggestions on 1st round review, would "加载中“ be better?